micrometer-core/src/main/java/io/micrometer/core/instrument/MeterRegistry.java

```diff
@@ -325,6 +325,8 @@ protected String getConventionName(Meter.Id id) {
      */
     protected abstract DistributionStatisticConfig defaultHistogramConfig();
 
+    // I don't know how to make NullAway understand the EnumMap can't return null
```

Comment:
Should we create an issue?

Reply:
Maybe. It's hard to tell sometimes what is intended behavior vs a bug vs a misunderstanding on my part with these things.